cmd/daemon.go

```diff
@@ -250,6 +250,12 @@ func (cd *CLIDaemon) UpdateJobStatus(jobID string) error {
 		return err
 	}
 
+	// 	- get job id (db) and worker id (?)
+	// - Subscribe to multiple channels
+	// - Count # of received messages for each channel
+	// - Compare to expected
+	// - pass or fail each channel
```

Comment:
comment artifacts?

Reply:
Yeah was for my own thought process haha good catch will remove